Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceSerializer.cpp

```diff
@@ -23,6 +23,53 @@ namespace AzToolsFramework
     {
         AZ_CLASS_ALLOCATOR_IMPL(JsonInstanceSerializer, AZ::SystemAllocator, 0);
 
+        namespace Internal
+        {
+            inline static const char* PathStartingWithEntities = "/Entities/";
+            inline static const char* PathMatchingContainerEntity = "/ContainerEntity";
```

Comment:
You should make this a `static constexpr AZStd::string_view` since you are taking the string length of these constant.
A `constexpr string_view` would perform `__builtin_strlen` over the literal at compile time, saving the runtime cost.

```suggestion
            static constexpr AZStd::string_view PathStartingWithEntities = "/Entities/";
            static constexpr AZStd::string_view PathMatchingContainerEntity = "/ContainerEntity";
```

Reply:
Just curious why you omitted the inline declaration here. Is it because its redundant?

Reply:
Yes it is redundant.
There is no need to inline the constants as it only ODR used in a cpp file and marking it as static gives the variable `internal` linkage.

This constants are not being compiled into multiple object files as they are not in a header, so the `inline` keyword is not needed.